site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-15-May-2022.adoc

```diff
@@ -0,0 +1,125 @@
+
+As a result, we are advising 3.0 users to upgrade to 3.0.26; 3.11 users to upgrade to 3.11.12; and 4.0 users should upgrade to 4.0.3.
+ 
+Please read the https://gitbox.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=NEWS.txt;hb=refs/tags/cassandra-4.0.3[release notes^] and https://issues.apache.org/jira/browse/CASSANDRA[let us know^] if you encounter any problems.
+
+See the xref:download.adoc[download section] for the latest stable and older supported versions of source and binary distributions.
+
+To stay up-to-date, we recommend joining the Cassandra xref:community.adoc#discussions[mailing lists].
+
+== Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._
+
+_For newcomers to the project, we have a useful xref:development/index.adoc[‘Contributing to Cassandra’] page for how to get involved and get started._
+
+_We also recommend taking a look at two Jira ticket queries we’ve created. One is a Kanban board for https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=496&quickFilter=2252[“Failing Tests” tickets that are unassigned^] and the other corresponds to our https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162&quickFilter=2160[Low Hanging Fruit or “Starter Tickets”^] for 4.0.x and 4.1.x. Feel free to self-select a ticket to work on._ 
+
+_Any of these tickets should be of appropriate complexity for someone new to the project to tackle. Just remember to assign yourself to the ticket and acknowledge the status, such as ‘Work in Progress’ and ‘Needs Comitter/Patch Available’ when you submit your patch. You can also reach out on the https://the-asf.slack.com/[ASF Slack^] in the #cassandra-dev Slack channel. Use @cassandra_mentors to contact our Cassandra mentors!_
+
+Read PMC member Josh McKenzie’s https://lists.apache.org/thread/lnmw2b4hzkl977p9hxv1sbdfqlq5snc6[latest bi-weekly update^] for ongoing discussions and the latest on ticket progress.
+
+=== Discussed
+
+As we mentioned, above, we forked Cassandra 4.1 and froze the branch on 1 May. The community has also agreed to block releases on green CI and are https://butler.cassandra.apache.org/#/[cleaning up test failures^] before release.
```

Comment:
Just making the dates consistent as `Mmm D` (instead of `D Mmm`):
```suggestion
As we mentioned, above, we forked Cassandra 4.1 and froze the branch on May 1. The community has also agreed to block releases on green CI and are https://butler.cassandra.apache.org/#/[cleaning up test failures^] before release.
```